app/config/drupal10-demo/install.sh

```diff
@@ -69,7 +69,7 @@ pushd "${CMS_ROOT}/sites/${DRUPAL_SITE_DIR}" >> /dev/null
   if [ -d "$CIVI_CORE/tools/extensions/org.civicrm.contactlayout" ]; then
     mv $CIVI_CORE/tools/extensions/org.civicrm.contactlayout files/civicrm/ext
   fi
-  cv api extension.refresh
+  cv api extension.refresh local=1
```

Comment:
Agree this idea seems fine.

When I first ran this, it still crashed on the remote download step. If you look at the 'api.default` for `Extension.refresh`, it specifies both `local=1` and `remote=1`.

The clearer version of this call would be like:

```suggestion
  cv api extension.refresh local=1 remote=0
```

That did fix for me.

(But when I went back to revalidate everything and understand the failing lookups... well, at that exact moment, the remote lookups seemed to work again...)

Anyway, I think it's a fine change since this script doesn't actually download extensions. So skipping the remote-refresh will trim some time+risk.

Reply:
Oops. Updated.

In terms of understanding the fails, I believe it's this https://lab.civicrm.org/infra/extdir/-/issues/9, which has been exacerbated by the recent-ish change to always include "unapproved" extensions.

And @stesi561 yes I believe it does pull it in here: https://github.com/civicrm/civicrm-buildkit/blob/af5c0ec662461353dad5cb9ddd34b2f87444875e/app/config/drupal10-demo/download.sh#L32

Reply:
aha! I should have hit grep!